### ISSUE TITLE
GitHub compare view seems to have gone away

### ISSUE THREAD

**Issue by alexose** [jameseisenhauer/ShotsOnSauce#2]
The image comparison feature no longer exists, which makes this otherwise excellent tool a lot less useful.

Unfortunately, this sort of belongs to GitHub, so there's not much to be done about it.  I did send them an email-- If I learn anything, I'll post it here.

**Comment by jimmyeisenhauer** [comment]
yeah and saucelabs no longer has a free account...just a free trial.  👎
